src/kirby/components/toolbar/toolbar.component.ts

```diff
@@ -19,6 +19,8 @@ export class ToolbarComponent implements OnInit {
   @Input() hideBackButton: boolean;
 
   @Output() back: EventEmitter<null> = new EventEmitter<null>();
```

Comment:
Remove obsolete generic type:
```suggestion
  @Output() back: EventEmitter = new EventEmitter();
```

Reply:
Cannot remove generic type this way without provoking lint. Has to be: 
@Output() back = new EventEmitter();

Reply:
Sorry my bad :)